### PR TITLE
Scripts to bring up a docker-based frankenstein setup

### PR DESCRIPTION
**Pull request by tomwilkie**

### CODE REVIEW

File: frankenstein/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM       quay.io/prometheus/busybox:latest
+COPY frank /bin/frank
```
Comment (juliusv) [inline frankenstein/Dockerfile]: Indentation is off here...

File: frankenstein/run.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+start_container 1 consul consul -p 8500:8500 -- agent -ui -server -client=0.0.0.0 -bootstrap
+
+# Services
+start_container 1 prometheus ingestor \
+    -v ${DIR}/ingestor-config:/etc/prometheus/
+start_container 1 frankenstein distributor \
+    -- -consul.hostname=consul1:8500
+
+# Tell distributor about ingestor
+curl -X PUT -d '{"hostname": "http://ingestor1:9090/push", "tokens": [0]}' "http://$(docker-machine ip $(docker-machine active)):8500/v1/kv/collectors/localhost"
```
Comment (juliusv) [inline frankenstein/run.sh:56]: Maybe just to be sure, sleep for 1s or so before this to ensure that Consul is ready?

**Comment by juliusv** [comment]
:+1:

File: frankenstein/run.sh (hunk 2 above)
Comment (tomwilkie) [inline frankenstein/run.sh:56]: Will do, although the new ingestor does this automatically, so it will go away soon.

**Comment by juliusv** [comment]
LGTM!

**Pull request merged**